.github/actions/integration-tests/action.yml

```diff
@@ -56,7 +48,7 @@ runs:
     - run: sudo ${{ github.action_path }}/setup-ci-endpoint --run-environment github --ci-endpoint ${{ inputs.ci_endpoint }}
       shell: bash
 
-    - run: sudo ${{ github.action_path }}/run-ci-stage1 --run-environment github --scenarios ${{ inputs.scenarios }} --userenvs ${{ inputs.userenvs }} --samples ${{ inputs.run_samples }} --repeat-runs ${{ inputs.repeat_runs }} --ci-endpoint ${{ inputs.ci_endpoint }} --ci-build-controller ${{ inputs.ci_build_controller }} --ci-param-mode ${{ inputs.ci_param_mode }}
```

Comment:
with this change I won't be able to run more samples for a given userenv ?

Reply:
It depends on which calling scenario you are asking about.  This is only affecting the github action usage that is called from a workflow.  If you are using the `crucible run-ci` interface then it is still possible to use the existing command line arguments to modify the samples and repeat run parameters.